src/scripts_install/@MakeLabelTable

```diff
@@ -1,10 +1,12 @@
 #!/bin/tcsh
+onintr END
 
 @global_parse `basename $0` "$*" ; if ($status) exit 0
 
+set tmpdir = ./
 set stat = 0
 set RNS = `3dnewid -fun`
```

Comment:
I wonder if 
```
set RNS = "tmp_mlt_`3dnewid -fun`"
```
might be slightly preferable here?  That way, there is some consistent start to the filename, before the random chars.